### PR TITLE
[EDU-1200] Add page for SDK setup instructions

### PR DESCRIPTION
**Pull request by m-hulbert**
## Description

This PR adds a page covering the setup instructions for SDKs. It combines the existing REST and realtime usage pages and adds more context related to API keys and ClientOptions.

See [JIRA](https://ably.atlassian.net/browse/EDU-1200) for further information.

## Review

View `/docs/getting-started/setup` to review this new page.

### CODE REVIEW

File: content/getting-started/setup.textile [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+      "context"
+      "fmt"
+      "github.com/ably/ably-go/ably"
+  )
+  ```
+
+blang[python].
+  The Python SDK is available from "PyPI":https://pypi.org/project/ably/. To get started with your project install the SDK using @pip@. Note that the Python SDK only implements the REST interface and also requires @asyncio@ to run.
+
+  Ably supports both string and binary payloads. To avoid ambiguity, it is recommended that strings passed to the library for publishing to Ably, for example, as an event name or payload data, should be unicode strings. In Python 3 this is the normal string type, but in Python 2 it is not. Ably suggests that you prefix string literals with the @u@ prefix and to explicitly decode any user input, such as @raw_input().decode(sys.stdin.encoding@.
```
Comment (tomczoink) [inline content/getting-started/setup.textile:153]: > @raw_input().decode(sys.stdin.encoding@

Missing a closing bracket.

File: content/getting-started/setup.textile [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+let rest = ARTRest(key: apiKey)
+```
+
+```[rest_flutter]
+final clientOptions = ably.ClientOptions(key: '<API_KEY>');
+ably.Rest rest = ably.Rest(options: clientOptions);
+```
+
+```[rest_go]
+client, err := ably.NewREST(ably.WithKey("xxx:xxx"))
```
Comment (tomczoink) [inline content/getting-started/setup.textile:276]: All other API key representations are with some form of 'api_key', should this be the same for consistency?

File: content/getting-started/setup.textile [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+if err != nil {
+        panic(err)
```
Comment (tomczoink) [inline content/getting-started/setup.textile:278]:
```suggestion
    panic(err)
```

**Review by tomczoink — CHANGES_REQUESTED** [review]

File: content/getting-started/setup.textile [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+  - ruby
+  - swift
+  - go
+---
+
+Use these instructions to setup an Ably SDK. "How to use Ably":/basics/use-ably outlines the differences between the realtime and REST interfaces in Ably SDKs and the other possible ways to use Ably.
+
+h2(#api). Get an API key
+
+An "API key":/auth#api-keys is required to instantiate a client and authenticate with Ably. "Sign up":https://ably.com/sign-up to Ably to create an API key in the "dashboard":https://ably.com/dashboard or use the "Control API":/account/control-api to create an API programmatically.
```
[the remark this answers is not in the document]
Comment (m-hulbert) [reply content/getting-started/setup.textile]: This is a good shout, as we should be point this out at every opportunity

File: content/getting-started/setup.textile [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+  ```[go]
+  import (
+      "context"
+      "fmt"
+      "github.com/ably/ably-go/ably"
+  )
+  ```
+
+blang[python].
+  The Python SDK is available from "PyPI":https://pypi.org/project/ably/. To get started with your project install the SDK using @pip@. Note that the Python SDK only implements the REST interface and also requires @asyncio@ to run.
```
[the remark this answers is not in the document]
Comment (m-hulbert) [reply content/getting-started/setup.textile:151]: I'll be updating everything for Python realtime separately, so have left it out of this one.

File: content/getting-started/setup.textile (hunk 2 above)
Comment (m-hulbert) [reply content/getting-started/setup.textile]: Updated and also fixed the realtime Go one.

**Review by tomczoink — APPROVED** [review]

**Pull request merged**